ports/libuuid/CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5)

```

```suggestion
cmake_minimum_required(VERSION 3.30)
```

If we're bothering to edit this we should choose the current value vcpkg requires:

https://github.com/microsoft/vcpkg/blob/ea2a964f9303270322cf3f2d51c265ba146c422d/scripts/vcpkg-tools.json#L26